### PR TITLE
Change storage blob track1 to track2

### PR DESCRIPTION
**Pull request by Luyunmt**
Change storage blob track1 to track2

### CODE REVIEW

**Comment by v-xuto** [comment]
@jongio - Can you help ping the corresponding person to review this PR?

**Comment by v-xuto** [comment]
@scottaddie, @AlexGhiondea, @kyle-patterson , @tg-msft, @amishra-dev - Could you help review this PR?

File: WebApp-Storage-DotNet/Controllers/HomeController.cs [hunk 1]
```diff
@@ -63,25 +61,24 @@ public async Task<ActionResult> Index()
...
-                blobContainer = blobClient.GetContainerReference(blobContainerName);
+                blobContainer = blobServiceClient.GetBlobContainerClient(blobContainerName);
                 await blobContainer.CreateIfNotExistsAsync();
 
                 // To view the uploaded blob in a browser, you have two options. The first option is to use a Shared Access Signature (SAS) token to delegate  
                 // access to the resource. See the documentation links at the top for more information on SAS. The second approach is to set permissions  
                 // to allow public access to blobs in this container. Comment the line below to not use this approach and to use SAS. Then you can view the image  
                 // using: https://[InsertYourStorageAccountNameHere].blob.core.windows.net/webappstoragedotnet-imagecontainer/FileName 
-                await blobContainer.SetPermissionsAsync(new BlobContainerPermissions { PublicAccess = BlobContainerPublicAccessType.Blob });
+                await blobContainer.SetAccessPolicyAsync(PublicAccessType.Blob);
```
Comment (tg-msft) [inline WebApp-Storage-DotNet/Controllers/HomeController.cs]: Let's just pass `PublicAccessType.Blob` into `CreateIfNotExistsAsync`.  It'll teach people better patterns.

File: WebApp-Storage-DotNet/Controllers/HomeController.cs [hunk 2]
```diff
@@ -63,25 +61,24 @@ public async Task<ActionResult> Index()
...
                 List<Uri> allBlobs = new List<Uri>();
-                foreach (IListBlobItem blob in blobContainer.ListBlobs())
+                foreach (BlobItem blob in blobContainer.GetBlobs())
                 {
-                    if (blob.GetType() == typeof(CloudBlockBlob))
-                        allBlobs.Add(blob.Uri);
+                    string a = blobContainer.Uri.ToString();
+                    if (blob.Properties.BlobType == BlobType.Block)
+                        allBlobs.Add(new Uri(blobContainer.Uri.ToString()+"/"+blob.Name));
                 }
```
Comment (tg-msft) [inline WebApp-Storage-DotNet/Controllers/HomeController.cs]: Do `blobContainer.GetBlob(blob.Name).Uri` instead of all this.

Comment (Tianyun-G) [reply WebApp-Storage-DotNet/Controllers/HomeController.cs]: Updated.

File: WebApp-Storage-DotNet/Controllers/HomeController.cs (hunk 1 above)
Comment (Tianyun-G) [reply WebApp-Storage-DotNet/Controllers/HomeController.cs]: @tg-msft - Updated.

**Comment by v-xuto** [comment]
@tg-msft  We have updated, please re-review this PR.

**Review by tg-msft — APPROVED** [review]

**Comment by v-xuto** [comment]
@tg-msft  Could you please help merge this PR?

**Pull request merged**